### PR TITLE
Add pad_value argument to make_grid and save_image

### PR DESCRIPTION
**Pull request by DmitryUlyanov**
This is useful, especially because 2 month ago the default color for padded pixels was white, while now I see black borders by default and cannot change it.

### CODE REVIEW

File: torchvision/utils.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 irange = range
 
 
-def make_grid(tensor, nrow=8, padding=2,
+def make_grid(tensor, nrow=8, padding=2, pad_value=0,
```
Comment (fmassa) [inline torchvision/utils.py]: While I prefer this order of arguments, do we want to possibly break backward compatibility by changing the order of arguments?
Also, the README is not automatically updated it seems. @soumith do we need to modify by hand both the inline documentation and the README?

Comment (soumith) [inline torchvision/utils.py]: please make pad_value to be the last argument.

Comment (DmitryUlyanov) [inline torchvision/utils.py]: done

**Comment by fmassa** [comment]
Could you also add the new parameter to `README.rst`? I don't think the README is autogenerated ATM...
After that, it's good to go!

**Comment by fmassa** [comment]
Thanks!

**Pull request merged**